content/management/changelog/index.mdx

```diff
@@ -51,6 +51,7 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method had the `webhook_id` parameter renamed to `id`.
 - The **customer_created** webhook was renamed to [**incoming_customer**](/management/configuration-api/v3.3/#incoming_customer).
+- The [**customer_session_fields_updated**](/management/configuration-api/v3.3/#customer_session_fields_updated) webhook was added.
```

Comment:
@bkubiak, do we want to mention new webhooks in the Agent Chat API and Customer Chat API Changelogs?

Reply:
I think that this update is strictly connected to v3.3 of configuration-api. 
Even if a dev calls `set-customer-session-fields` on v3.1 in customer-api, it will result in the webhook, too. So we shouldn't assign the info to the v3.3 changelog of customer-api as it affects all versions in the service. cc @czeczot